3ds/flac/PKGBUILD

```diff
@@ -0,0 +1,47 @@
+# Maintainer:  Dave Murphy <davem@devkitpro.org>
+# Contributor: cpasjuste <cpasjuste@gmail.com>
+# Contributor: oreo639 <oreo6391@gmail.com>
+
+pkgname=3ds-flac
+pkgver=1.3.2
+pkgrel=1
+pkgdesc='Open source Free Lossless Audio Codec'
+arch=('any')
+url='https://xiph.org/flac/'
+license=(Xiph.org)
+options=(!strip libtool staticlibs)
+depends=("3ds-libogg")
+makedepends=('devkitpro-pkgbuild-helpers')
+groups=('3ds-portlibs')
+
+source=(
+  "https://ftp.osuosl.org/pub/xiph/releases/flac/flac-$pkgver.tar.xz"
+  "flac-${pkgver}.patch"
+)
+sha256sums=(
+  '91cfc3ed61dc40f47f050a109b08610667d73477af6ef36dcad31c31a4a8d53f'
+  'SKIP'
+)
+
+build() {
+  cd flac-$pkgver
+
+  source /opt/devkitpro/3dsvars.sh
+
+  patch -Np1 -i "$srcdir/flac-${pkgver}.patch"
+
+  CFLAGS="${CFLAGS} -D_3DS " \
```

Comment:
-D_3DS is included in CPPFLAGS here - https://github.com/devkitPro/pacman-packages/blob/master/pkgbuild-scripts/3dsvars.sh#L8 can you check if it's working?

Reply:
Yeah, it works.